src/seedu/addressbook/ui/TextUi.java

```diff
@@ -76,44 +65,44 @@ private boolean isCommentLine(String rawInputLine) {
      * @return command (full line) entered by the user
      */
     public String getUserCommand() {
-        out.print(LINE_PREFIX + "Enter command: ");
+        String promptCommand = Formatter.getUserCommandFormat(PROMPT_ENTER_COMMAND);
+        out.print(promptCommand);
         String fullInputLine = in.nextLine();
 
         // silently consume all ignored lines
         while (shouldIgnore(fullInputLine)) {
             fullInputLine = in.nextLine();
         }
-
-        showToUser("[Command entered:" + fullInputLine + "]");
+        
+        String echoCommand = Formatter.echoCommandFormat(fullInputLine);
+        showToUser(echoCommand);
```

Comment:
You have both `showToUser()` and `out.print()` here. They are essentially doing the same thing. Consider which one we should use at this level.